nuget/readme.md

```diff
@@ -2,6 +2,72 @@
 
 ## Overview
 
-Please visit [Microsoft.Windows.CsWinRT](https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/) for official Microsoft-signed builds of the NuGet package. 
+Please visit [Microsoft.Windows.CsWinRT](https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/) for official Microsoft-signed builds of the NuGet package.
+
+The Microsoft.Windows.CsWinRT NuGet package provides WinRT projection support for the C# language.  Adding a reference to this package causes C# projection sources to be generated for any Windows metadata referenced by the project.  For details on package use, please consult the Microsoft.Windows.CsWinRT github repo's [readme.md](https://github.com/microsoft/cswinrt/blob/master/README.md).
+
+C#/WinRT detects Windows metadata from:
+* Platform winmd files in the SDK 
+* NuGet package references containing winmd files
+* Other project references producing winmd files
+* Raw winmd file references
+
+For any winmd file discovered above, C#/WinRT creates reference (consuming) projection sources (*.cs files).  These sources can then be compiled into a projection assembly for client projects to reference, or compiled directly into an app.
```

Comment:
This line is confusing to me. How does this sound:

For any winmd file discovered above, C#/WinRT creates reference projection sources. These sources are C# files that can be consumed. And then compiled into a projection assembly...

Reply:
it sounds better, but it misapplies the term "consumed" - it's not about what we do with the C# files - it's the kind of projection (consuming) they implement.  I'll fiddle with what you have...